### DISCUSSION TITLE
Cut a release

### DISCUSSION THREAD

**Discussion by chenrui333** [gokrazy/gokrazy#212] [Q&A]
👋 any interest of cutting a release for distribution? Thanks!

**Comment by stapelberg** [comment]
Typically, users should keep their gokrazy systems up-to-date, which should include the gokrazy/gokrazy main repository that goes into the installation and the gokrazy/tools repository for the packer. 

Given that they are all supposed to be run from their respective latest commit, I’m not sure there is much value in a tagged release. We don’t have defined support windows or semantics around version numbers, either.

Which distribution are you asking for?
